stingray/simulator/simulator.py

```diff
@@ -419,13 +419,7 @@ def _simulate_model(self, model):
         pos_real   = self.random_state.normal(size=nbins//2)*fac
         pos_imag   = self.random_state.normal(size=nbins//2)*fac
 
-        pos_freq_transform = pos_real + 1j * pos_imag
-
-        # Simulate light curve from its Fourier transform
-        arg  = np.concatenate(([self.mean], pos_freq_transform))
-
-        # Inverse Fourier transform
-        long_lc = np.fft.irfft(arg)
+        long_lc = self._find_inverse(pos_real, pos_imag)
```

Comment:
@usmanwardag do you see any unwanted behavior that might come out of this change?